src/Arcus.WebApi.Tests.Integration/Logging/Controllers/EchoController.cs

```diff
@@ -20,5 +25,12 @@ public IActionResult Post([FromBody] string body)
         {
             return Ok(body);
         }
+
+        [HttpPost]
+        [Route("devices/{deviceId}/echo")]
+        public IActionResult Post([FromRoute] int deviceId, [FromBody] string body)
+        {
+            return Ok(body);
+        }
```

Comment:
I'm not sure if we should pollute this simple controller with specific routes. Maybe we can create a separate API controller for this to keep this one clean and simple for other integration tests to use.

Reply:
I can do that, but don't know if it is necessary to introduce another controller, just to have an operation with route-params.

Reply:
We have several one-method controllers, all to keep the functionality and the test fixtures clear and not to pollute each other as little as possible. In this case, we could improve this since the `EchoController` is only about having a simple controller without 'complex' stuff like route parameters.

Reply:
If the a simple controller like the `EchoController` is enough for the test, we can leave it 👍 . My concern was most of all about altering the `EchoController` for the sake of the test.